src/Compiler/CompilerUtils.hpp

```diff
@@ -42,9 +42,10 @@ void setTargetArch(const std::string &arch);
 void setTargetTriple(const std::string &triple);
 void setOptLevel(const onnx_mlir::OptLevel level);
 // Set compile context according to the (key, value) fields passed.
-void setCompileContext(mlir::MLIRContext &context,
-    const llvm::SmallVector<std::pair<onnx_mlir::OptionKind, std::string>, 4>
-        options);
+using CompilerOptionList =
+    llvm::SmallVector<std::pair<onnx_mlir::OptionKind, std::string>, 4>;
+void setCompileContext(
+    mlir::MLIRContext &context, const CompilerOptionList &options);
 // Set compile context, legacy C array and string representation.
 void setCompileContext(mlir::MLIRContext &context,
```

Comment:
If you are referring to this call
```
void setCompileContext(mlir::MLIRContext &context,
    const onnx_mlir::OptionKind *key, const char **val, const int64_t num);
```

it is needed because we want to support a C interface as well.

Reply:
ok good.